core/scripts/testing/scripts.js

```diff
@@ -14,6 +14,26 @@
     document.head.appendChild(style);
   }
 
+  /**
+   * The term `palette` is used to as a param to match the
+   * Ionic docs, plus here is already a `ionic:theme` query being
+   * used for `md`, `ios`, and `ionic` themes.
+   */
+  const palette = window.location.search.match(/palette=([a-z]+)/);
+  if (palette && palette[1] !== 'light') {
+    const linkTag = document.createElement('link');
+    linkTag.setAttribute('rel', 'stylesheet');
+    linkTag.setAttribute('type', 'text/css');
+    /**
+     * The file must be the same as the one being used in Playwright tests
+     * when `dark` is being passed as a `theme` option.
+     * This provides consistency when Playwright is running through
+     * the test page or the `setContent` method.
+     */
+    linkTag.setAttribute('href', `/scripts/testing/themes/${palette[1]}.css`);
```

Comment:
Do we plan to update this for v8? https://github.com/ionic-team/ionic-framework/tree/feature-8.0/core/src/css/palettes

Reply:
I've already updated it here since v8 came out first.